plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/mylyn/tasks/ui/view/TaskRepositoriesView.java

```diff
@@ -90,7 +90,7 @@ public void activateTask (String taskName) {
 		DefaultShell shellThatTakesLongTimeToClose = new DefaultShell("Activate Task");
 		new DefaultText().setText(taskName);
 		new PushButton("OK").click();	
-		new WaitWhile(new ShellIsAvailable(shellThatTakesLongTimeToClose), TimePeriod.LONG);
+		new WaitWhile(new ShellIsAvailable(shellThatTakesLongTimeToClose), TimePeriod.VERY_LONG);
```

Comment:
Please overload this method with additional parameter TimePeriod timeout. So you will be able to set on your own time out in your tests even if it goes higher next time. So you would not have to create a new PR to prolong timeout.